selfdrive/car/gm/interface.py

```diff
@@ -57,14 +48,17 @@ def sig(val):
     # An important thing to consider is that the slope at 0 should be > 0 (ideally >1)
     # This has big effect on the stability about 0 (noise when going straight)
     # ToDo: To generalize to other GMs, explore tanh function as the nonlinear
-    a, b, c, _ = [2.6531724862969748, 1.0, 0.1919764879840985, 0.009054123646805178]  # weights computed offline
+    if self.CP.carFingerprint == CAR.BOLT_EUV:
+      a, b, c, _ = [2.6531724862969748, 1.0, 0.1919764879840985, 0.009054123646805178]  # weights computed offline
+    elif self.CP.carFingerprint == CAR.ACADIA:
+      a, b, c, _ = [4.78003305, 1.0, 0.3122, 0.05591772]
 
     steer_torque = (sig(lateral_accel_value * a) * b) + (lateral_accel_value * c)
     return float(steer_torque) + friction
 
   def torque_from_lateral_accel(self) -> TorqueFromLateralAccelCallbackType:
-    if self.CP.carFingerprint == CAR.BOLT_EUV:
-      return self.torque_from_lateral_accel_bolt
+    if self.CP.carFingerprint in [CAR.BOLT_EUV, CAR.ACADIA]:
```

Comment:
this list is in two places. suggest moving the a, b, c weights to a dict at the top and using the dict keys here